.github/workflows/integration_test.yaml

```diff
@@ -13,3 +13,4 @@ jobs:
       chaos-enabled: true
       chaos-experiments: pod-delete
       chaos-app-label: app.kubernetes.io/name=discourse-k8s
+      chaos-duration: 360
```

Comment:
I suppose you tested locally to define this value ?
It will probably depends on the perf of the machine running the test, did you add some "buffer" when setting this value ?

Reply:
I did a basic test of bringing up a clean microk8s cluster, installing the discourse charm with required relations and deleting the discourse pod, observing what will happen (and when). This was the local test, so not using Litmus Chaos directly - I thought that wouldn't be needed, however the test fails again and I need to see how to reproduce it, perhaps via mimicking the CI/CD - cloning the repo with experiments, setting environment variables and observing what happens. It's a tedious process I did when initially troubleshooting Litmus for the Indico charm and hoped I could skip that part. Alas.

Reply:
Strange that it failed after 244s, the error doesn't explicitely states it's a timeout, so the issue could be different